<launch>
  <!-- Arguments -->
  <arg name="new_frame_id" default="robot"/>
  <arg name="robot_names" default="['base_link_23']"/>
  <arg name="reference_robot_name" default="base_link_40"/>
  <arg name="reference_robot_rotation" default="0.0"/>
  <arg name="input_bag_file" default=""/>

  <!-- Node launching -->
  <node pkg="mercator_sensor_fusion_ros_pkg" type="ground_truth_publisher" name="ground_truth_publisher" output="screen">
    <!-- Parameters -->
    <param name="new_frame_id" value="$(arg new_frame_id)" />
    <param name="robot_names" value="$(arg robot_names)" />
    <param name="reference_robot_name" value="$(arg reference_robot_name)" />
    <param name="reference_robot_rotation" value="$(arg reference_robot_rotation)" />
    <param name="input_bag_file" value="$(arg input_bag_file)" />

  </node>
</launch>

